src/plugins/kibana_react/public/exit_full_screen_button/exit_full_screen_button.tsx

```diff
@@ -54,8 +51,7 @@ class ExitFullScreenButtonUi extends PureComponent<ExitFullScreenButtonProps> {
           </p>
         </EuiScreenReaderOnly>
         <div className="dshExitFullScreenButton">
-          <KuiButton
-            type="hollow"
+          <button
```

Comment:
Should this be `EuiButton` instead?

Reply:
I haven't pulled the code locally, it looks from the class definition there are some style hacks that may not be necessary anymore if this becomes EUI component:
https://github.com/elastic/kibana/blob/85d4147cfd85a1eba343ac619e4d2b5ec667a3ee/src/plugins/kibana_react/public/exit_full_screen_button/_exit_full_screen_button.scss#L9

Reply:
Hey @majagrubic thank you for your comments! 

When it comes to the first one, this button is pretty special and EuiButton adds the styles that are not needed here. If we want to use our EUI components only and not native html components, I can overwrite more styles, but I cannot think of a reason why it should be EuiButton. 

When it comes to the styles, they actually are needed for the button to animate, stay in fixed position and behave as expected, I am not sure why comment states it's a hack 🤔 

Attaching the gif of the button for clarity. 

 
![button](https://user-images.githubusercontent.com/4283304/69712231-c4591280-1102-11ea-9201-aca2d337b4f9.gif)